jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js

```diff
@@ -70,7 +71,14 @@ YUI.add('deployment-signup', function() {
     */
     _handleSignup: function() {
       this._setBetaCookie(true);
-      console.error('Not implemented.');
+      this.props.changeState({
```

Comment:
I guess it could be confusing but the alternative is to leave them on the beta sign up page and make them find the button above or close and reopen the deployment. UX was happy with it working this way.